src/i18n/en.js

```diff
@@ -159,12 +159,12 @@ You'll find an Email with steps to follow.`,
     download_msg: `Download`,
     downloads_column: `Downloads`,
     due_date_column: `Due Date`,
-    error_msg: `There was an error gathering the invoices.`,
-    no_data_msg: `There are no invoices to show`,
-    no_download_msg: `There is no invoice to download`,
+    error_msg: `Ouch! An error occurred while trying to upload your invoices.`,
+    no_data_msg: `There are no issued invoices to show yet.`,
+    no_download_msg: `There is no invoice to download yet.`,
     paid_to_date_column: `Paid`,
     product_column: `Product`,
-    sub_title: `List of issued invoices`,
+    sub_title: `Here is the detail of the invoices we have issued. You may download each one.`,
```

Comment:
How about this?
You may download each one. --> Remember you can download each one of them individually.

Sounds more natural to me.